pkg/networkmanager/firewall-client.js

```diff
@@ -204,20 +204,24 @@ function initFirewalldDbus() {
 
 firewalld_service.addEventListener('changed', () => {
     const installed = !!firewalld_service.exists;
+    const is_running = firewalld_service.state === 'running';
+
+    // we get lots of these events, for internal property changes; filter interesting changes
```

Comment:
Hmm, should this filtering happen in pkg/lib/service.js maybe? But other code might rely on getting events also when service.unit changes...

Reply:
Is this maybe a change in behavior introduced by the python bridge? Maybe the C bridge would batch change notifications better?

Reply:
We can't filter them in service.js, as the `firewalld_service` object exposes all systemd properties, and it can't know which ones the client is interested in. It could be that the C bridge handled this differently and merged multiple  D-Bus `PropertyChanged` signals into one handler call. I somehow doubt it, as that is a heuristic which is hard to do right for every use case, but I'll check this on a c8s build.

Reply:
I added a console.log to the handler and print the `.state` and `.exists` fields. On a `centos-8-stream` build it does not look different, there are also bursts of signal handlers:
```
-> wait: ph_is_present("#networking-firewall-summary input[type=checkbox]:checked")
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true

-> ph_mouse("#networking-firewall-summary input[type=checkbox]:not([disabled]):not([aria-disabled=true])","click",0,0,0,false,false,false,false)
> log: XXX firewalld_service.changed stopped true
> log: XXX firewalld_service.changed stopped true
> log: XXX firewalld_service.changed stopped true
> log: XXX firewalld_service.changed stopped true
> log: XXX firewalld_service.changed stopped true
> log: XXX firewalld_service.changed starting true
> log: XXX firewalld_service.changed starting true
> log: XXX firewalld_service.changed starting true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
> log: XXX firewalld_service.changed running true
```

I'm not even sure if this flake is a regression with the Python bridge. It's not *that* common, we may have had it for a long time (just now I am putting a lot of attention to unstable tests).